combirepo/rpm_patcher.py

```diff
@@ -359,7 +361,7 @@ def __install_rpmrebuild(self, queue):
         @param queue    The queue where the result will be put.
         """
         make_command = ["sudo", "chroot", self.patching_root, "bash", "-c",
-                        """cd /rpmrebuild/src; chmod +x /usr/bin/make; make; make install"""]
+                        """cd /rpmrebuild/src; make; make install"""]
```

Comment:
I think `;` is better to replace with `&&`, so we won't get half-built version installed

Reply:
Done